android-components/components/browser/engine-gecko/build.gradle

```diff
@@ -89,4 +90,20 @@ dependencies {
 apply plugin: "org.mozilla.telemetry.glean-gradle-plugin"
 apply from: '../../../android-lint.gradle'
 apply from: '../../../publish.gradle'
+apply plugin: "org.mozilla.appservices.nimbus-gradle-plugin"
+nimbus {
+    // The path to the Nimbus feature manifest file
+    manifestFile = "geckoview.fml.yaml"
```

Comment:
I wonder if we should have `nimbus.fml.yaml`, which applies to the whole AC and not only to the `engine-gecko` module, @jhugman what do you think?

Reply:
No, for a number of reasons:

- the generated code is per app, and not every app needs every component. You could use a FML component per app, and share FML files by including them in to each of the app's component, however…
- the generated code needs to be accessible from the component itself. If you had a per-app FML component, then the components (e.g. `engine-gecko`) would need to know which app it's in at compile time.

I have thought about having a per app FML file in AC, which is just `imports` of the component FML files, which then gets included by the apps: this might be work; I'd suggest thinking about this refactor when the third component grows an FML file.